conans/client/manager.py

```diff
@@ -185,7 +185,8 @@ def _get_graph_builder(self, loader, update, remote_proxy):
         return graph_builder
 
     def _get_deps_graph(self, reference, profile, filename, current_path, remote_proxy):
-        loader = ConanFileLoader(self._runner, self._client_cache.settings, self._profile_with_defaults(profile))
+        loader = ConanFileLoader(self._runner, self._client_cache.settings,
+                                 self._profile_with_defaults(profile), self._user_io.out)
```

Comment:
You need the real settings, not the profile settings values. What if you want to remove the `os` in the preprocessing? https://github.com/conan-io/conan/pull/1565